app/route.php

```diff
@@ -7,4 +7,5 @@
 $router->addRoute('POST', '/contract/delete', 'App\Controllers\ApiController::deleteContract');
 $router->addRoute('POST', '/contract/delete/metadata', 'App\Controllers\ApiController::deleteContractMetadata');
 $router->addRoute('POST', '/contract/delete/text', 'App\Controllers\ApiController::deleteContractText');
-$router->addRoute('POST', '/contract/delete/annotation', 'App\Controllers\ApiController::deleteContractAnnotation');
\ No newline at end of file
+$router->addRoute('POST', '/contract/delete/annotation', 'App\Controllers\ApiController::deleteContractAnnotation');
+$router->addRoute('POST', '/contract/published_at/update', 'App\Controllers\ApiController::updatePublishedAtIndex');
```

Comment:
url should not contain underscore use dash instead of underscore

Reply:
It will be used only once